hedera-node/src/main/java/com/hedera/services/store/contracts/AbstractLedgerWorldUpdater.java

```diff
@@ -156,9 +154,7 @@ public Account get(final Address addressOrAlias) {
 	}
 
 	@Override
-	public EvmAccount getAccount(final Address addressOrAlias) {
-		final var address = aliases().resolveForEvm(addressOrAlias);
```

Comment:
Why do we need to remove this line? I think we should keep it or we may brake other logic like create2

Reply:
That logic has been moved in the [overridden method `HederaStackedWorldStateUpdater`](https://github.com/hashgraph/hedera-services/pull/3152/files#diff-61c1dade85468a15389978645ebb3c22afd195d0ecab11af92df6e0f86e39e42R153-R155)